plugins/connection/httpapi.py

```diff
@@ -266,6 +280,10 @@ def send(self, path, data, retries=None, **kwargs):
             use_proxy=self.get_option("use_proxy"),
             headers={},
         )
+        # Only insert ciphers kwarg for ansible-core versions >= 2.14.0
+        if StrictVersion(ANSIBLE_CORE_VERSION) >= StrictVersion("2.14.0"):
+            url_kwargs["ciphers"] = self.get_option("ciphers")
```

Comment:
```suggestion
            url_kwargs["ciphers"] = self.get_option("ciphers")
        else if self.get_option("ciphers"):
            self.queue_message(
                "warning",
                "'ciphers' option is unavailable on ansible-core<2.14",
            )
```

Reply:
I have implemented a variation of this. I think the warning to the user should be for the full "ansible_httpapi_ciphers" since that is the variable they see, ok?